### ISSUE TITLE
4.2.15 breaks building docs

### ISSUE THREAD

**Issue by linusha** [ankitskvmdam/clean-jsdoc-theme#259]
**Describe the bug**

Since today, so with the release of 4.2.15 our pipeline building our docs fails. The only change comes due to us using the newest version of this package, as we had not pinned it earlier. Using 4.2.14 resolved our problem.

This problem occurs in a GitHub Actions hosted runner on linux. A failing run can be found [here](https://github.com/LivelyKernel/lively.next/actions/runs/6787966551/job/18452122200#step:3:1).

The following error occurs:

```
undefined:13
for (const script of this.includeScript) {
                          ^

TypeError: this.includeScript is not iterable
    at Template.eval (eval at template (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/node_modules/underscore/underscore-node-f.cjs:924:14), <anonymous>:13:27)
    at Template.template (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/node_modules/underscore/underscore-node-f.cjs:931:19)
    at Template.partial (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/lib/jsdoc/template.js:55:33)
    at Template.eval (eval at template (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/node_modules/underscore/underscore-node-f.cjs:924:14), <anonymous>:288:14)
    at Template.template (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/node_modules/underscore/underscore-node-f.cjs:931:19)
    at Template.partial (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/lib/jsdoc/template.js:55:33)
    at Template.render (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/lib/jsdoc/template.js:69:28)
    at generate (/home/runner/work/lively.next/lively.next/node_modules/clean-jsdoc-theme/publish.js:332:17)
    at /home/runner/work/lively.next/lively.next/node_modules/clean-jsdoc-theme/publish.js:369:9
    at Array.forEach (<anonymous>)
    at generateSourceFiles (/home/runner/work/lively.next/lively.next/node_modules/clean-jsdoc-theme/publish.js:344:30)
    at exports.publish (/home/runner/work/lively.next/lively.next/node_modules/clean-jsdoc-theme/publish.js:917:9)
    at module.exports.cli.generateDocs (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/cli.js:441:39)
    at module.exports.cli.processParseResults (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/cli.js:392:24)
    at module.exports.cli.main (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/cli.js:235:18)
    at module.exports.cli.runCommand (/opt/hostedtoolcache/node/18.12.1/x64/lib/node_modules/jsdoc/cli.js:186:9)
```

**Comment by gdelacampa** [comment]
Beat me to it am getting this same issue as well. Rolled back to 4.2.12 in package.json and it started working again.

**Comment by ankitskvmdam** [comment]
Thanks @linusha and @gdelacampa for reporting this bug. 
Working on this Bug,

**Comment by ankitskvmdam** [comment]
I have published `v4.2.16`, it will fix the build issue. 🤞
